roles/openshift_health_checker/library/aos_version.py

```diff
@@ -27,7 +25,12 @@
 try:
     import yum  # pylint: disable=import-error
 except ImportError as err:
-    IMPORT_EXCEPTION = err  # in tox test env, yum import fails
+    IMPORT_EXCEPTION = err
+
+
+LIST_PKG_NAME = 0
+LIST_PKG_VERSION = 1
+LIST_PKG_MULTI_MINOR_CHECK = 2
```

Comment:
I really do not like non-named tuples as an API to this module. Could it be a list of dict instead? It would be clearer on both sides to see:

               dict(
                    name="docker",
                    version=self.get_required_docker_version(task_vars),
                    check_multi=False,
                ),

and

    p['name']